Source/WebCore/html/HTMLTemplateElement.idl

```diff
@@ -39,4 +39,5 @@
     [CEReactions=NotNeeded, Reflect=shadowrootclonable] attribute boolean shadowRootClonable;
     [EnabledBySetting=DeclarativeShadowRootsSerializerAPIsEnabled, CEReactions=NotNeeded, Reflect=shadowrootserializable] attribute boolean shadowRootSerializable;
     [EnabledBySetting=ScopedCustomElementRegistryEnabled, CEReactions=NotNeeded, Reflect=shadowrootcustomelements] attribute boolean shadowRootCustomElements;
+    [CEReactions=NotNeeded, EnabledBySetting=ShadowRootReferenceTargetEnabled] attribute [AtomString] DOMString shadowRootReferenceTarget;
```

Comment:
Why not just add `Reflect=shadowrootreferencetarget` instead of writing setter & getter manually?